src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -2984,6 +2984,12 @@
   <data name="InvalidOperation_SpanOverlappedOperation" xml:space="preserve">
     <value>This operation is invalid on overlapping buffers.</value>
   </data>
+  <data name="InvalidOperation_TimeProviderNullLocalTimeZone" xml:space="preserve">
+    <value>The operation cannot performed when having TimeProvider.LocalTimeZone is returning null.</value>
```

Comment:
> The operation cannot performed when having TimeProvider.LocalTimeZone is returning null.

Shouldn't this be something closer to:
> The operation cannot be performed when TimeProvider.LocalTimeZone is  null.

Similarly, all the others feel slightly wrong.

Reply:
Thanks. I fixed the messages.